extension/copy-gist.js

```diff
@@ -0,0 +1,24 @@
+/* globals utils */
+
+'use strict';
+
+window.addGistCopyButton = () => {
+	// Button already added (partial page nav), or non-text file
+	if ($('.copy-btn').length > 0) {
+		return;
+	}
+	const $gistsSibling = $('.file-actions > .btn.btn-sm');
+
+	for (const anchor of $gistsSibling) {
+		const gistUri = anchor.href;
+		if (gistUri.split('.').pop().toLowerCase() !== 'md') {
```

Comment:
There are more file types that get rendered than just Markdown, like structured text. That's why I asked for detecting when it's not a source code view.